packages/bpk-component-datatable/package.json

```diff
@@ -19,7 +19,7 @@
     "bpk-react-utils": "^7.0.0",
     "lodash": "^4.17.20",
```

Comment:
Looks like with these changes `lodash` is no longer being used? So we can remove it as a dep?

Reply:
Yess!!!! 🎉